sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs

```diff
@@ -14,7 +14,12 @@ namespace Azure.AI.FormRecognizer.Samples
     public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
     {
         /// <summary>
-        /// This sample illustrates how to consume Bounding Box type data that is available across the Form Recognizer library.
+        /// This sample illustrates how to consume a Bounding Box type data that is available across the Form Recognizer library.
+        /// A bounding box is a sequence of four <see cref="PointF"/> representing a quadrilateral that outlines
+        /// the text of an element in a recognized form.
```

Comment:
nit: in v2.1 there may be non-textual elements - maybe say and element, such as a line or a word, ... ?

Reply:
thank you! I was not aware